apps/btc_family/btc_fam_pub_key.c

```diff
@@ -62,8 +62,12 @@
 
 #include "bip32.h"
 #include "btc.h"
-#include "btc_api.h"
-#include "btc_helpers.h"
+#include "btc/core.pb.h"
+#include "btc_fam_api.h"
+#include "btc_fam_context.h"
+#include "btc_fam_helpers.h"
+#include "btc_fam_priv.h"
+#include "pb_encode.h"
```

Comment:
Can you please double check includes - since there are a lot of new files, there could be circular includes as well..